README.md

```diff
@@ -1,2 +1,24 @@
 # cap
 The Common Alerting Protocol (CAP) allows emergency messages to be simultaneously disseminated over a wide variety of existing and emerging public alerting systems.
+
+## Project Purpose
+This project is for providing client libraries, and examples, in one or more somewhat popular languages for container hosted services (rust, golang, js, swift, etc.).
+
+## Call for Code
+This is a [Call for Code](https://callforcode.org/) (C4C) project for other C4C projects to build on. C4C is a multi-year global initiative asking developers to use their skills and mastery of the latest technologies, to drive positive change across the world through their code. 
+<p align="center">
+<a href="https://callforcode.org/"><img src="https://callforcode.org/wp-content/uploads/2018/05/Call_for_Code_logo_vector_toptype_color.png" alt="Call for Code" width="120" height="138" border="10"></a>
+</p>
+
+## CAP Description
+CAP is an international technical data specification developed by the Organization for the Advancement of Structured Information Standards (OASIS).
+* In addition to the basic CAP standard, a supplemental IPAWS Profile technical specification was developed to ensure compatibility with existing warning systems used in the United States.
+* FEMA has formally adopted CAP and the IPAWS Profile to implement the Integrated Public Alert and Warning System (IPAWS).
+For a detailed description see [OASIS CAP Specification]( https://www.fema.gov/media-library/assets/documents/105521).
+
+## Communication
+To communicate with the team please feel free to open up issues and pull requests on this github repo.
+
+## Licenses
+The cap codebase is released under the [Apache 2.0 license](https://github.com/IBM/cap/LICENSE).
+The README.md file, and files in the "docs" folder are licensed under the [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
```

Comment:
I'd just use Apache II for both - the split might confused people and I'm not sure its worth the discussion.

Reply:
will leave per chat...

Reply:
how bizzare